cache-config/traffic_ops_ort.pl

```diff
@@ -102,6 +103,23 @@
 if ( defined $mode ) {
 	$cmd .= ' --run-mode=' . $mode;
 }
+
+if ( defined( $ARGV[1] ) ) {
+	my $log_level = "";
+        my $ort_log_level = uc($ARGV[1]);
+	given ( $ort_log_level ) {
+                when ("ALL")   { $log_level = "-vv"; }
+                when ("TRACE") { $log_level = "-vv"; }
+                when ("DEBUG") { $log_level = "-vv"; }
+                when ("INFO")  { $log_level = "-v"; }
+                when ("WARN")  { $log_level = ""; }
```

Comment:
`-v` means warn, `-vv` means info. I think this line should be `-v` and the above should be `-vv`.

Reply:
Ok. I can update that. Without tc3-apply working in report, it's hard to know what should be expected.